File: KingdomCodeTool/wwwroot/Download/202410291354/Angular/shared/DanhMucPhongBan.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucPhongBan } from 'src/app/shared/DanhMucPhongBan.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucPhongBanService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucPhongBan";
    }
}

File: KingdomCodeTool/wwwroot/Download/202410291354/Service/Implement/DanhMucPhongBanService.cs
namespace Service.Implement
{
    public class DanhMucPhongBanService : BaseService<DanhMucPhongBan, IDanhMucPhongBanRepository>
    , IDanhMucPhongBanService
    {
    private readonly IDanhMucPhongBanRepository _DanhMucPhongBanRepository;
    public DanhMucPhongBanService(IDanhMucPhongBanRepository DanhMucPhongBanRepository) : base(DanhMucPhongBanRepository)
    {
    _DanhMucPhongBanRepository = DanhMucPhongBanRepository;
    }
    }
    }

